oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -328,6 +318,47 @@ void CodeUpdate::setVersions()
                         "xyz.openbmc_project.Software.Activation";
                     auto imageObjPath = path.str.c_str();
 
+                    // If the version interface is added with the Activation
+                    // value as Invalid, it is considered as assemble code
+                    // update image failure Failure is occured during assembling
+                    // lid files are creating a tar ball.
```

Comment:
nit: period after "update image failure". 
      "lid files and creating a tar ball"